### ISSUE TITLE
steam_await not available?

### ISSUE THREAD

**Issue by biesnecker** [facebook/hhvm#6626]
Trying to use the async stream_await function, but it doesn't seem to be available (nor do its associated contents. `hh_client` tells me this:

```
app/WatchmanClient.php:26:21,32: Unbound name (typing): stream_await (Typing[4107])
app/WatchmanClient.php:26:21,32: Unbound name: stream_await (a global constant) (Naming[2049])
app/WatchmanClient.php:26:21,32: Unbound name: stream_await (a global function) (Naming[2049])
app/WatchmanClient.php:26:49,66: Unbound global constant (Typing) (Typing[4106])
app/WatchmanClient.php:26:49,66: Unbound name: STREAM_AWAIT_WRITE (a global constant) (Naming[2049])
app/WatchmanClient.php:28:19,36: Unbound global constant (Typing) (Typing[4106])
app/WatchmanClient.php:28:19,36: Unbound global constant (Typing) (Typing[4106])
app/WatchmanClient.php:28:19,36: Unbound global constant (Typing) (Typing[4106])
app/WatchmanClient.php:28:19,36: Unbound name: STREAM_AWAIT_READY (a global constant) (Naming[2049])
```

(not sure what the duplication that's going on there is)

And actually running the code gives me this:

```
Catchable fatal error: Hack type error: Unbound name (typing): stream_await at /Users/jbiesnecker/Projects/app/WatchmanClient.php line 26
```

I'm on OSX 10.10.5, and `hhvm --version` looks like this:

```
HipHop VM 3.10.1 (rel)
Compiler: 1449068303_N
Repo schema: 09e5c37b8f5a97d4efed8ef28ceae9252cdfba36
```

**Comment by steelbrain** [comment]
I am able to repro this on the latest version

**Comment by fredemmott** [comment]
Already fixed in nightlies and 3.11: https://github.com/facebook/hhvm/issues/5748
